.circleci/config.yml

```diff
@@ -284,3 +294,19 @@ jobs:
           command: |
             sudo npm install -g markdown-link-check
             bash .circleci/check-links/check-links.sh
+
+  windows-msi:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - attach_to_workspace
+      - run:
+          name: Install Wix Toolset
+          command: .\scripts\make.ps1 Install-Tools
+      - run:
+          name: Build MSI
+          command: .\scripts\make.ps1 New-MSI
```

Comment:
@james-bebbington Maybe we can add a simple `msiexec` installation test step in this job after building the msi?  Or better yet, we can add a new job that would test the installation, service start/stop, and uninstallation.

Reply:
I included this validation check as part of the same `windows-msi` job. It does feel like it would be nice to do this in another job, but that would require another `attach_to_workspace` step, etc, which is quite slow.